### ISSUE TITLE
2.0.0-beta4:  rate() method looking for undefined sound._sprite

### ISSUE THREAD

**Issue by daltonridenhour** [goldfire/howler.js#387]
In the rate() method, sound._sprite is undefined when the howl is managing a single audio file:
```javascript
var duration = ((self._sprite[sound._sprite][0] + self._sprite[sound._sprite][1]) / 1000) - seek;
```

I used this to fix for my purposes, but likely breaks with audio sprites:
```javascript
var duration = ((self._sprite['__default'][0] + self._sprite['__default'][1]) / 1000) - seek;
```

**Comment by goldfire** [comment]
Can you give a code example? It should never be the case that there is no sprite defined, even if you are playing a single sound. The `__default` sprite should always get setup, and so far I haven't been able to cause that to break.

**Comment by daltonridenhour** [comment]
https://codepen.io/anon/pen/RWEqey

Try this.  I should be able to call _howl.rate(0.5), yes?

**Comment by goldfire** [comment]
Ah, I was always calling `rate` after `play`. This should now be fixed in 73dfb18e43c46b0e08cac284dd4a577a713ad09f.
